src/common/api-endpoints.ts

```diff
@@ -29,4 +29,11 @@ export const endpoints = {
     recurringPrices: <Endpoint>{ url: '/donation/prices/recurring', method: 'GET' },
     createCheckoutSession: <Endpoint>{ url: '/donation/create-checkout-session', method: 'POST' },
   },
+  bootcamp: {
+    bootcampList: <Endpoint>{ url: '/bootcamp', method: 'GET' },
+    getIntern: <Endpoint>{ url: '/bootcamp', method: 'GET' },
```

Comment:
Normally when doing a get you need to provide `id` or `slug` in the URL of the request. 

You might check how the `campaign.viewCampaign` method provides a function to build the URL. this is well typed way of handling the proper params and allows easy refactoring.

```tsx
viewCampaign: (slug: string) => <Endpoint>{ url: `/campaign/${slug}`, method: 'GET' },
```